webpages/styles/components/search.css

```diff
@@ -1,12 +1,10 @@
 .search-box {
-  background: var(--search-toolbar-background);
```

Comment:
I'm not so sure about this color change Since it doesn't scroll the same way as he background.

Reply:
idk i didnt add the color change, git is getting confused ig

Reply:
also there is a box-shadow to differentiate